kernel/kernel-impl/src/main/java/org/sakaiproject/content/impl/FileConversionServiceImpl.java

```diff
@@ -28,39 +28,47 @@
 import java.util.concurrent.ScheduledExecutorService;
 import java.util.concurrent.TimeUnit;
 
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.criterion.Restrictions;
+
 import org.sakaiproject.authz.api.SecurityAdvisor;
 import org.sakaiproject.authz.api.SecurityService;
 import org.sakaiproject.component.api.ServerConfigurationService;
 import org.sakaiproject.content.api.ContentHostingService;
 import org.sakaiproject.content.api.ContentResource;
 import org.sakaiproject.content.api.FileConversionService;
-import org.sakaiproject.content.api.repository.FileConversionQueueItemRepository;
 import org.sakaiproject.content.hbm.FileConversionQueueItem;
 import org.sakaiproject.content.impl.converters.LoolFileConverter;
+import org.sakaiproject.db.api.SqlService;
 import org.sakaiproject.entity.api.ResourceProperties;
 import org.sakaiproject.entity.api.ResourcePropertiesEdit;
 import org.sakaiproject.exception.IdUnusedException;
 import org.sakaiproject.exception.PermissionException;
-import org.springframework.transaction.TransactionStatus;
-import org.springframework.transaction.annotation.Transactional;
-import org.springframework.transaction.support.TransactionCallbackWithoutResult;
-import org.springframework.transaction.support.TransactionTemplate;
 
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.transaction.support.TransactionTemplate;
 
 import lombok.Setter;
 import lombok.extern.slf4j.Slf4j;
 
+import javax.annotation.Resource;
+
 @Slf4j
 @Setter
 public class FileConversionServiceImpl implements FileConversionService {
 
     @Autowired private ContentHostingService contentHostingService;
-    @Autowired private FileConversionQueueItemRepository repository;
     @Autowired private SecurityService securityService;
     @Autowired private ServerConfigurationService serverConfigurationService;
+    @Autowired private SqlService sqlService;
```

Comment:
You can do straight up sql using hibernate no need to bring in sqlservice

Reply:
I used sql service as I didn't want to load the entity before updating it with a simple update. Can you do that? Also, there's very little testing you can do with this. The main thing that needs testing is the actual libre office integration, ie: load testing. It seemed real overkill to create a Dao just for these simple calls when there'd be little to no yield in unit testing.

Reply:
You can evict an Entity after making changes.

Reply:
Changes made. All hibernate now. I still think it seems overkill to add a dao interface and impl, though. You can't even test this meaningfully without bringing up a docker container with libre in it.